src/main/kotlin/li/cli/oc/Components.kt

```diff
@@ -33,10 +39,10 @@ object Components {
         Cable("cable", Cable()),
         Capacitor("capacitor", Capacitor()),
         CarpatedCapacitor("carpetedcapacitor", CarpetedCapacitor()),
-        CaseOne("case1", Case(1)),
-        CaseTwo("case2", Case(2)),
-        CaseThree("case3", Case(3)),
-        CaseCreative("casecreative", Case(4)),
+        CaseOne("case1", Case(1, FabricBlockSettings.of(Material.METAL).nonOpaque())),
+        CaseTwo("case2", Case(2, FabricBlockSettings.of(Material.METAL).nonOpaque())),
+        CaseThree("case3", Case(3, FabricBlockSettings.of(Material.METAL).nonOpaque())),
```

Comment:
Why was this changed? The block settings are the same on the Cases

Reply:
I'll be honest, I don't precisely remember. I think it came about from needing to access the `settings` in Case.kt before they were passed on to TecBlock? I'd need to do a bit more digging to be sure.

Reply:
in Kotlin there is a init function so you dont need to pass it

Reply:
Yeah, I have no idea why I thought I needed this. It works fine the other way. I'll push a fix.

Reply:
cool